_topic_maps/_topic_map_rosa.yml

```diff
@@ -112,6 +112,12 @@ Topics:
   File: cloud-experts-dynamic-certificate-custom-domain
 - Name: Assigning consistent egress IP for external traffic
   File: cloud-experts-consistent-egress-ip
+- Name: Getting started with ROSA
```

Comment:
Are there going to be more tutorials that are located in the "Getting started with ROSA" subheader? If not, then the Setup page should be located under the Tutorials heading.